### ISSUE TITLE
Persisting the background service after device restart

### ISSUE THREAD

**Issue by AnthonySmith01** [NativeScript/sample-ios-background-execution#2]
Hi

Thanks for this great project.

Quick question, is there anyway to make this background service trigger or persist after a full device restart?

**Comment by ivanbuhov** [comment]
Hi @sm1ddy 
In the iOS world it is hard (someone may say impossible) to trigger a background task on boot. Technically there is no such API or mechanism but some workarounds exist. Keep in mind that these restrictions exist for native iOS applications too. You can find more in-depth information in [this SO answer](http://stackoverflow.com/a/20086364/3968054).

**Comment by AnthonySmith01** [comment]
Thanks mate, much appreciated.

Update (for others looking at this):
My app relies on location services so I got around the "reboot" issue by using the significant location change api. Basically, the significant change api wakes the app whenever the device has moved about 500m (according to Apple). This also persists across reboots. 

I made a simple/rough solution by modifying the nativescript-geolocation module as mentioned in this issue:
https://github.com/NativeScript/nativescript-geolocation/issues/44
